### PR TITLE
Remove font smoothing for non-retina

### PR DESCRIPTION
**Pull request by AmeeMorris**
Our good friend font-smoothing again.

Maybe this will be the final cull

### CODE REVIEW

**Comment by AmeeMorris** [comment]
[Issue: Settings menu love](https://trello.com/c/g0OYd4d1/3834-issue-settings-menu-love)

File: styles/overrides/_popover.scss [hunk 1]
```diff
@@ -2,9 +2,11 @@
   font-family: $typo-stack-base;
   box-shadow: $shadow-base;
   border-color: $neutral-light;
-  @include font-smoothing;
   animation: dropdownAnimation 750ms;
   animation-fill-mode: forwards;
+  @media (-webkit-min-device-pixel-ratio: 1.25), (min-resolution: 120dpi) {
```
Comment (kyleharper) [inline styles/overrides/_popover.scss]: This is added via the global styling. I think we just need to remove `@include font-smoothing;` from the existing code 👍

File: styles/overrides/_tooltip.scss [hunk 2]
```diff
@@ -1,8 +1,10 @@
 .tooltip {
     font-family: $typo-stack-base;
-    @include font-smoothing;
     animation: dropdownAnimation 750ms;
     animation-fill-mode: forwards;
+    @media (-webkit-min-device-pixel-ratio: 1.25), (min-resolution: 120dpi) {
```
Comment (kyleharper) [inline styles/overrides/_tooltip.scss]: This is added via the global styling. I think we just need to remove `@include font-smoothing;` from the existing code 👍

**Comment by AmeeMorris** [comment]
@kyleharper done

**Pull request merged**